helpers/repositories.json

```diff
@@ -344,6 +344,7 @@
    "botinflux/mexc-dnx-bot:latest",
    "blocxtech",
    "spectrachrome",
+   "aapanel/aapanel:lib"
```

Comment:
This is missing a comma at end of line.

Reply:
I fixed it
27ab755f434b32edf36c09992b92c602488d1bc0